backend/swift/swift.go

```diff
@@ -201,6 +201,11 @@ func init() {
 It should be set to true for resuming uploads across different sessions.`,
 			Default:  false,
 			Advanced: true,
+		}, {
+			Name:     "directory_markers",
+			Default:  false,
+			Advanced: true,
+			Help:     `If true, create directory markers with the content type application/directory for Mkdir requests.`,
```

Comment:
Is this the swift way? Should the markers end with trailing / ? Be zero sized?

Reply:
Yes, it is. `application/directory` is the preferred content type and the trailing slash is not required - https://github.com/openstack/python-swiftclient/blob/e65070964c7b1e04119c87e5f344d39358780d18/swiftclient/service.py#L213 .

It's also worth mentioning that Swift treats `b`, `b/`, `b//`, and `b///`  as different objects, so I think it's better to just remove all trailing slashes to prevent conflicts. 

This simple go program creates `a/b`, `a/b/` and `a/b//` under a container `foo` https://gist.github.com/joec4i/446e2cc4deb9bc276dc8fdb37559f7a9 . 

The swift CLI shows them as 3 different objects.
```
$ swift -A http://127.0.0.1:8080/auth/v1.0 -U test:tester -K testing list foo
a/b
a/b/
a/b//
```